Day6/jsDates.js

```diff
@@ -0,0 +1,20 @@
+// The days of the week are: "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"
+function getDayName(dateString) {
+  let dayName;
+  dayName = [
```

Comment:
Why not just decalre the dayNames variable straight forward. I think doing that will keep your code clean and simple.